apps/onboarding/src/subsidy/subsidy.service.ts

```diff
@@ -33,14 +33,16 @@ export class SubsidyService {
       return walletValid
     }
 
-    const attestations = await this.contractKit.contracts.getAttestations()
+
     const res = await this.txParserService.parse(
       input.requestTx,
-      input.walletAddress, {
-        [attestations.address]: {
-          [attestations.methodIds.request]: true
-        }
-    })
+      input.walletAddress,
+      new MethodFilter().addContract(
+        CeloContract.Accounts,
```

Comment:
Should this be `CeloContract.Attestations`?

Reply:
It should! Gosh darn-it I couldn't find a good way to extract the contract name from the wrapper (or at least a typing link between the enum and wrapper) without going back and making changes in contractkit, and it didn't feel worth it. But there ya go.